backend/src/main.rs

```diff
@@ -22,7 +22,6 @@ mod sync;
 mod util;
 
 mod build_info {
-    include!(concat!(env!("OUT_DIR"), "/built.rs"));
 }
```

Comment:
Even weirder, here you forgot to remove the `mod` thingy so there is another module here?! What :D How are even semantics?!

Reply:
Whoops :sweat_smile: